openproblems/data/tnbc_wu2021.py

```diff
@@ -0,0 +1,75 @@
+from . import utils
+
+import numpy as np
+import os
+import scanpy as sc
+import scipy.sparse
+import scprep
+import tempfile
+
+URL = "https://figshare.com/ndownloader/files/36590871"
```

Comment:
Can you please add a comment linking to the figshare.com/articles/... URL that describes this figshare dataset?